### ISSUE TITLE
Some JAR files will be empty

### ISSUE THREAD

**Issue by maskit** [apache/incubator-pulsar#685]
#### Expected behavior

No warnings during build

#### Actual behavior

```
[INFO] --- maven-jar-plugin:3.0.0:test-jar (default) @ pulsar ---
[WARNING] JAR will be empty - no content was marked for inclusion!
[INFO] Building jar: /Users/masakazu/git/pulsar/target/pulsar-tests.jar
```

```
[INFO] --- maven-jar-plugin:3.0.0:test-jar (default) @ distribution ---
[WARNING] JAR will be empty - no content was marked for inclusion!
[INFO] Building jar: /Users/masakazu/git/pulsar/all/target/distribution-tests.jar
```

#### Steps to reproduce
```
mvn install -DskipTests
```

#### System configuration
**Pulsar version**: master c51eaad
**OS**: macOS 10.12

**Comment by maskit** [comment]
#678 fixes this.

**Comment by merlimat** [comment]
Merged #678
